src/firebase/logger.ts

```diff
@@ -122,22 +141,28 @@ export class Logger {
     }
   }
 
-  // Sets the localStorage key with the given suffix (queuedKeySuffix or
-  // sendingKeySuffix) to contain the supplied records.
+  // Sets the localStorage item identified by |keySuffix| (|queuedKeySuffix| or
+  // |sendingKeySuffix|) to contain |records|.
   _setRecords(keySuffix: string, records: LogRecord[], prefix?: string) {
     const key = (prefix || this._storagePrefix) + keySuffix;
     localStorage.setItem(key, JSON.stringify(records));
   }
 
-  // Adds the supplied records to the queue.
+  // Adds |records| to the end of the queue.
   _enqueueRecords(records: LogRecord[]) {
     const queued = this._getRecords(queuedKeySuffix);
     queued.push(...records);
     this._setRecords(queuedKeySuffix, queued);
   }
 
-  // Returns true if records are in-flight to the Cloud Function.
-  _currentlySending(): boolean {
+  // Returns true if a call to _sendQueued() is scheduled. Exposed so that tests
+  // can verify that the class is idle.
+  sendScheduled() {
```

Comment:
please rename to IsSendScheduled. I thought this actually sent all the scheduled messages.

Reply:
done